src/main/webapp/pipe.js

```diff
@@ -331,105 +338,119 @@ function getLink(data, link) {
-                html.push('</tr>');
-            });
-            html.push('</tbody>');
-            html.push('</table>');
+        html.push('<a href=' + getLink(data,analysis.url) + '>' + analysis.name + '</a>');
+        html.push('<table id="priority.summary" class="pane">');
+        html.push('<tbody>');
+        html.push('<tr>');
+        html.push('<td class="pane-header">Total</td>');
+        html.push('<td class="pane-header">Failures</td>');
+        html.push('<td class="pane-header">Skipped</td>');
+        html.push('</tr>');
+        html.push('</tbody>');
+        html.push('<tbody>');
+        html.push('<tr>');
+        html.push('<td class="pane">' + analysis.total + '</td>');
+        html.push('<td class="pane">' + analysis.failed + '</td>');
+        html.push('<td class="pane">' + analysis.skipped + '</td>');
+        html.push('</tr>');
+        html.push('</tbody>');
+        html.push('</table>');
         html.push('</div></div>');
-        html.push('</div>');
-        return html.join('');
+    });
+
+    html.push('</div>');
+    return html.join('');
+}
+
+function generateStaticAnalysisInfo(data, task) {
+    if (!data.showStaticAnalysisResults || !task.staticAnalysisResults || task.staticAnalysisResults.length === 0) {
+        return undefined;
     }
+
+    var html = ['<div class="infoPanelOuter">'];
+    html.push('<div class="infoPanel"><div class="infoPanelInner">');
+    html.push('<table id="priority.summary" class="pane">');
+    html.push('<thead>');
+    html.push('<tr>');
+    html.push('<td class="pane-header">Warnings</td>');
+    html.push('<td class="pane-header" style="font-size: smaller; vertical-align: bottom">High</td>');
+    html.push('<td class="pane-header" style="font-size: smaller; vertical-align: bottom">Normal</td>');
+    html.push('<td class="pane-header" style="font-size: smaller; vertical-align: bottom">Low</td>');
+    html.push('</tr>');
+    html.push('</thead>');
+    html.push('<tbody>');
+
+    Q.each(task.staticAnalysisResults, function(i, analysis) {
+        html.push('<tr>');
+            html.push('<td class="pane"><a href=' + getLink(data,analysis.url) + '>' + trimWarningsFromString(analysis.name) + '</a></td>');
+            html.push('<td class="pane" style="text-align: center">' + analysis.high + '</td>');
+            html.push('<td class="pane" style="text-align: center">' + analysis.normal + '</td>');
+            html.push('<td class="pane" style="text-align: center">' + analysis.low + '</td>');
+        html.push('</tr>');
+    });
+
+    html.push('</tbody>');
+    html.push('</table>');
+    html.push('</div></div>');
+    html.push('</div>');
+    return html.join('');
 }
 
 function trimWarningsFromString(label) {
     var offset = label.indexOf('Warnings');
-    return offset == -1 ? label : label.substring(0, offset).trim();
+    return offset === -1 ? label : label.substring(0, offset).trim();
 }
 
 function generatePromotionsInfo(data, task) {
```

Comment:
This method has same problem with big diff and same changes as in previous comment